opentelekomcloud/services/rds/resource_opentelekomcloud_rds_instance_v3.go

```diff
@@ -774,10 +775,14 @@ func resourceRdsInstanceV3Read(_ context.Context, d *schema.ResourceData, meta i
 		node["availability_zone"] = nodeObj.AvailabilityZone
 		node["status"] = nodeObj.Status
 		nodesList = append(nodesList, node)
+		availabilityZones = append(availabilityZones, nodeObj.AvailabilityZone)
```

Comment:
Order is vital here. Primary instance should be the first one.

Reply:
it comes in ordered list.
1st is master
2nd is slave
Should I sort them explicitly?

Reply:
I suppose, yes. I don't trust in order in server response :)